Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -440,4 +429,48 @@ def check_for_duplicate_pull_requests(formula)
       EOS
     end
   end
+
+  def alias_update_pair(formula, new_formula_version)
+    versioned_alias = formula.aliases.grep(/.@\d/).first
+    return if versioned_alias.nil?
+
+    name, old_alias_version = versioned_alias.split("@")
+    if old_alias_version.split(".").length == 1
```

Comment:
`new_alias_regex = if old_alias` ... and then assign just the regex (`/^\d+/` or `/^\d+\.\d+/`) to a variable to reduce a bit of duplication.

Reply:
Ah, that was what you've meant. I've forgot that ruby has an inline `if`/`else` too (however with quite a different syntax compared to python).